src/GridTilemap/GridTilemap.test.ts

```diff
@@ -376,6 +377,17 @@ describe("GridTilemap", () => {
     expect(mockCharBlockCache.removeCharacter).toHaveBeenCalledWith(charMock1);
   });
 
+  it("should find characters", () => {
+    gridTilemap = new GridTilemap(tilemapMock);
+
+    const charMocks = new Set<GridCharacter>();
+    charMocks.add(createCharMock("player"));
+    mockCharBlockCache.findCharactersAt = jest.fn(() => charMocks);
+
+    const set = gridTilemap.getCharactersAt(new Vector2(1, 1), "layer1");
```

Comment:
Same as above: could you add an expectation that emnsures that findCharactersAt has been called with the correct arguments?